bin/ci/test.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -o xtrace
+set -o errexit
+
+# Set to 'true' to run the known "manual" tests in rippled.
+MANUAL_TESTS=${MANUAL_TESTS:-false}
+# The maximum number of concurrent tests.
+CONCURRENT_TESTS=${CONCURRENT_TESTS:-$(nproc)}
+# The path to rippled.
+RIPPLED=${RIPPLED:-build/rippled}
+# Additional arguments to rippled.
+RIPPLED_ARGS=${RIPPLED_ARGS:-}
+
+function join_by { local IFS="$1"; shift; echo "$*"; }
+
+declare -a manual_tests=(
+  'beast.chrono.abstract_clock'
```

Comment:
we now have this list in two places - that's unfortunate.

Reply:
I agree. I'd like to have a well-documented pair of scripts, `build.sh` and `test.sh`, and replace `build-and-test.sh` with calls to them, but I need some help understanding `build-and-test.sh`. I don't have experience with everything it is using, and the parts I'm unfamiliar with are undocumented or underdocumented.